dessia_common/workflow/utils.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu July 7 14:52:32 2022
+
+@author: xavier-ferry
+"""
+
+from typing import List, Dict
+
+
+class ToScriptElement:
+    """
+    Class meant to improve to_script readability
+    """
+    before_declaration: str = None
```

Comment:
I think there is a misunderstanding on the differences between dessia_common and js here.
In python, attributes defined here are class attributes and its not the same concept that JS's private attributes.

Not only, you don't need to define an instance attribute here under the class definition, it is also wrong if you do it to attach the attribute to the class and change its value depending on the instance.

For example, if a Car has a class attribute for the number of wheels, which is 4 for every cars, a Ferrari instance (let's say the F1) **cannot** overwrite its value based on the instance, which is what is done here